tests/test_chmod_command.rs

```diff
@@ -0,0 +1,191 @@
+// Copyright 2023 The Jujutsu Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::path::Path;
+
+use crate::common::TestEnvironment;
+
+pub mod common;
+
+fn create_commit(
```

Comment:
We have similar functions (same?) in a few other places. Is it time to move it to a shared place? Maybe onto `TestEnvironment`?

Reply:
There seems to be two versions in general usage: one with files and another without. I can try to unify them.

Reply:
I might postpone this to another PR, and possibly for later. Cleaning this up would create merge conflicts.

BTW, do we like the `get_branch_output` function? On one hand, it could also move to `TestEnvironment`. On the other hand, we could just inline it everywhere.

Reply:
> I might postpone this to another PR, and possibly for later. Cleaning this up would create merge conflicts.

Sure, no problem.

> BTW, do we like the `get_branch_output` function? On one hand, it could also move to `TestEnvironment`. On the other hand, we could just inline it everywhere.

I would just inline it since it's so simple, and so it's clear what it does (it's easy to think that it does something more complex when it has its own function).